services/api/mw2pdf/package.json

```diff
@@ -19,5 +19,12 @@
     "pdfmake": "^0.1.68",
     "puppeteer": "^5.5.0",
     "uuid": "^8.3.2"
+  },
+  "devDependencies": {
+    "awesome-typescript-loader": "^5.2.1",
```

Comment:
I don't know webpack too well but wanted to sanity check: it looks like `awesome-typescript-loader` is a fork of `ts-loader` with about 10% of the user base (300k vs 3.2m).

More importantly: the awesome variety's last push is 3 years ago whereas ts-loader was 9 days ago.

If you actively picked this one understood and I trust your judgement!  If it was just a "this was the one that I've used in the past" then it might be worth considering using [ts-loader](https://www.npmjs.com/package/ts-loader) instead.
I just wanted to make sure we want to use that fork for whatever reason.

Reply:
I think that I decided to use it for exactly 1 reason -- that I used it in a previous project and I was cargo culting. Let's go with ts-loader at your suggestion. I agree that going with the mainstream is a good idea! Thanks for doing this research and checking me on it!

Reply:
Resolved in an updated version of the PR. Again, thanks!!